README.md

```diff
@@ -45,75 +45,76 @@ See unit test in xt folder for more examples and some use cases for specific API
 # POTENTIALLY SUPPORTED APIs
```

Comment:
This change was made by last night on devel branch. See https://github.com/sdondley/moo-google/commit/e532b8e8fad5f8b4c235810a4ffadc6368023615

Also, no changes should be made directly to README files. They are generated automatically by dist zilla.